plugins/linear-genome-view/src/LinearGenomeView/components/AboutDialog.tsx

```diff
@@ -75,14 +243,21 @@ export default function AboutDialog({
     >
       <DialogTitle id="alert-dialog-title">{trackName}</DialogTitle>
       <DialogContent>
-        {Object.entries(data || {}).map(([key, value]) => (
-          <div key={`${key}_${String(value)}`} className={classes.field}>
-            <div className={classes.fieldName}>{key}</div>
-            <div className={classes.fieldValue}>
-              <SanitizedHTML html={value} />
-            </div>
-          </div>
-        ))}
+        <BaseCard title="Configuration">
+          <Attributes attributes={data} />
+        </BaseCard>
+        {info !== null ? (
+          <BaseCard title="File info">
+            {info === undefined ? (
+              'Loading file data...'
+            ) : (
+              <Attributes attributes={info} showHidden={showHidden} />
+            )}
+            <Button onClick={() => setShowHidden(hidden => !hidden)}>
+              {!showHidden ? 'Show hidden entries' : 'Hide hidden entries'}
+            </Button>
```

Comment:
Any way to only show this button if there are hidden entries?

Reply:
Probably yes :)